components/ImageInput/constants.ts

```diff
@@ -1,2 +1,2 @@
-export const DEFAULT_HEIGHT = 150;
-export const DEFAULT_WIDTH = 300;
+export const DEFAULT_HEIGHT = "150px";
+export const DEFAULT_WIDTH = "300px";
```

Comment:
I don't think this change is necessary as numeric units get converted to pixels by React by default